data_quality/tasks/check_datapackage.py

```diff
@@ -15,8 +15,11 @@ class DataPackageChecker(Task):
 
     """A task runner to check that the data package is correct"""
 
-    def __init__(self, config):
+    def __init__(self, config, inflexible_resources=[]):
```

Comment:
It means it's not subject to change of schema. I thought I should call it "uncustomizable"  but it was too long. Feel free to propose something else.